src/main/java/com/uniqueauction/web/review/ReviewController.java

```diff
@@ -0,0 +1,38 @@
+package com.uniqueauction.web.review;
+
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.uniqueauction.domain.review.service.ReviewRepository;
+import com.uniqueauction.domain.review.service.ReviewService;
+import com.uniqueauction.web.response.CommonResponse;
+import com.uniqueauction.web.review.request.SaveReviewRequest;
+
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@RestController
+public class ReviewController {
+
+	private final ReviewService reviewService;
+	private final ReviewRepository reviewRepository;
+
+	@GetMapping("/review")
+	public CommonResponse<?> saveReview(SaveReviewRequest saveReviewRequest, BindingResult result) {
+		reviewService.save(saveReviewRequest);
+		return CommonResponse.success();
+	}
+
+	@PostMapping("/review/productRevies")
```

Comment:
- 오타가 있습니다.
- 상품에 대한 리뷰를 조회하는 api 인것 같습니다. `get`으로 변경하는게 적절합니다
```suggestion
	@GetMapping("/review/productReviews")
```

Reply:
헉 수정하겠습니다